rskj-core/src/main/resources/reference.conf

```diff
@@ -3,6 +3,20 @@ blockchain.config.hardforkActivationHeights.papyrus200 = -1
 
```

Comment:
reference.conf is used by convention :
https://github.com/lightbend/config#standard-behavior

Reply:
Thanks @patogalla